cmd/crio/cli.go

```diff
@@ -0,0 +1,571 @@
+package main
```

Comment:
I like that! Do we want to move this file into an internal `internal/pkg/cli` package? This would have the benefit that we could write tests for it later on.